client/extensions/woocommerce/app/settings/email/mailchimp/mailchimp_dashboard.js

```diff
@@ -0,0 +1,249 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
+import Button from 'components/button';
+import Card from 'components/card';
+import FormCheckbox from 'components/forms/form-checkbox';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormLegend from 'components/forms/form-legend';
+import FormToggle from 'components/forms/form-toggle';
+import FormRadio from 'components/forms/form-radio';
+import FormTextInput from 'components/forms/form-text-input';
+import Notice from 'components/notice';
+import QueryMailChimpSyncStatus from 'woocommerce/state/sites/settings/email/querySyncStatus';
+import {
+	syncStatus,
+	mailchimpSettings,
+	isRequestingSettings
+	} from 'woocommerce/state/sites/settings/email/selectors';
+import { submitMailchimpNewsletterSettings, requestResync } from 'woocommerce/state/sites/settings/email/actions.js';
+import { isSubmittingNewsletterSetting, newsletterSettingsSubmitError } from 'woocommerce/state/sites/settings/email/selectors';
+import { errorNotice, successNotice } from 'state/notices/actions';
+
+const SyncTab = localize( ( { siteId, translate, syncState, resync } ) => {
+	const { account_name, store_syncing, product_count, mailchimp_total_products,
+		mailchimp_total_orders, order_count } = syncState;
+	const hasProductInfo = ( product_count !== undefined ) && ( mailchimp_total_products !== undefined );
+	const products = hasProductInfo ? ( product_count + '/' + mailchimp_total_products ) : '';
+	const hasOrdersInfo = ( order_count !== undefined ) && ( mailchimp_total_orders !== undefined );
+	const orders = hasOrdersInfo ? ( order_count + '/' + mailchimp_total_orders ) : '';
+
+	const synced = () => (
+		<Notice
+			status="is-success"
+			isCompact
+			showDismiss={ false }
+			text={ translate( '%(mailingListname)s list synced.', {
+				args: { mailingListname: syncState.mailchimp_list_name } } ) }
+		/>
+	);
+
+	const syncing = () => (
+		<Notice
+			status="is-warning"
+			isCompact
+			showDismiss={ false }
+			text={ translate( '%(mailingListname)s list is being synced.', {
+				args: { mailingListname: syncState.mailchimp_list_name } } ) }
+		/>
+	);
+
+	const onResyncClick = () => {
+		! store_syncing && resync( siteId );
+	};
+
+	return (
+		<div>
+			<div>
+				<span className="mailchimp__account-info">{ translate( 'MailChimp account:' ) }</span>
```

Comment:
Recommend using the args approach instead here, like you did above, instead of breaking `MailChimp account:` and the `account_name` like this. RTL languages especially come to mind here